selfdrive/car/hyundai/carstate.py

```diff
@@ -208,7 +208,7 @@ def update_canfd(self, cp, cp_cam):
 
     # TODO: alt signal usage may be described by cp.vl['BLINKERS']['USE_ALT_LAMP']
     left_blinker_sig, right_blinker_sig = "LEFT_LAMP", "RIGHT_LAMP"
-    if self.CP.carFingerprint == CAR.HYUNDAI_KONA_EV_2ND_GEN:
+    if bool(cp.vl['BLINKERS']['USE_ALT_LAMP']):
```

Comment:
can you make this a tuple check for the two cars? don't add a flag yet, as once we detect it, it won't require a flag